sources/packages/web/src/components/students/StudentDocumentList.vue

```diff
@@ -12,8 +12,8 @@
           @click="fileUtils.downloadStudentDocument(ministryDocument)"
         >
           <span class="mr-4">
-            <font-awesome-icon :icon="['far', 'file-alt']"
-          /></span>
+            <v-icon icon="fa:far fa-file-alt" size="20"></v-icon
```

Comment:
Do we have a consistent icon size or are there different sizes in figma. In the other comment I made, the size is 18.

Reply:
all the places where I have added size for the icon were big than the Figma, for eg.. 
1. below is the icon with default size
![image](https://user-images.githubusercontent.com/77353155/182709768-c11fc1bd-6ca7-4846-969e-2f4ea9ac76bd.png)
below is the figma,
![image](https://user-images.githubusercontent.com/77353155/182709985-8336880f-9998-4383-bd2f-cc4a83b7b28a.png)

Reply:
cmmented above